src/omero_cli_render.py

```diff
@@ -49,11 +51,11 @@
     bin/omero render info RenderingDef:1
     bin/omero render info Image:123
 
-    # %(EDIT)s
-    bin/omero render edit Image:1 <YAML or JSON file>
-    where the input file contains a top-level channels key (required), and
-    an optional top-level greyscale key (True: greyscale, False: colour).
-    Channel elements are index:dictionaries of the form:
+    # %(SET)s
+    bin/omero render set Image:1 <YAML or JSON file>
+    # where the input file contains a top-level channels key (required), and
+    # an optional top-level greyscale key (True: greyscale, False: colour).
+    # Channel elements are index:dictionaries of the form:
 
     channels:
       <index>: (Channel-index, int, 1-based)
```

Comment:
Maybe ``int`` -> ``(int)`` to be in line with the examples ``(float)`` below https://github.com/ome/omero-cli-render/pull/7/files#diff-7c3731557868b1b97ef87421dd10cf92L62

Reply:
Also, possibly, the ``(int)`` should be after the ``1-based``